### ISSUE TITLE
Error when running i2cssh with only a cluster name as an option on CLI

### ISSUE THREAD

**Issue by Bazerghi** [wouterdebie/i2cssh#112]
**Describe the bug**
I have a valid .i2csshrc file that I've been using for years. This file also works perfectly if I use the -c or --clusters command-line parameters. For example the following works perfectly:

i2cssh -c MyCluster
i2cssh -c MyCluster1,MyCluster2

If I attempt to run i2cssh with only a single cluster name as an option, I get the error pasted later in this issue:

i2cssh MyCluster
i2cssh MyCluster2

**My config**
My .i2csshrc file is rather large and contains hostnames I would prefer not to share. A simple .i2csshrc file can be used to trigger the error:

```
---
version: 2
clusters:
  MyCluster:
    hosts:
       - host1
       - host2
   MyCluster1:
     hosts:
        - host1
        - host2
   MyCluster2:
     hosts:
        - host1
        - host2
```

As mentioned in the description above, the error is triggered if I try to use a cluster name without the -c option. 

**Expected behavior**

When a single cluster name is used on the commandline (without the -c option), i2cssh should connect to all the hosts defined in that cluster in the .i2csshrc file.

i2csshrc MyCluster          (This should be valid syntax)

**Screenshots or stack traces**

Traceback (most recent call last):
  File "/opt/homebrew/bin/i2cssh", line 450, in <module>
    main()
  File "/opt/homebrew/bin/i2cssh", line 446, in main
    i2cssh(prog_name="i2cssh")
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/opt/homebrew/bin/i2cssh", line 127, in i2cssh
    groups = [{
  File "/opt/homebrew/bin/i2cssh", line 129, in <listcomp>
    "geometry": compute_geometry(len(hosts), hosts[0].get("rows"), hosts[0].get("columns")),
AttributeError: 'list' object has no attribute 'get'

**Additional context**

Using just a single cluster name on the CLI when running i2cssh was functioning properly under the Ruby codebase version. Additionally, the 'Usage' displayed by i2cssh does indicate that this is valid syntax. 

Also, for reference - I initially installed i2cssh using Brew and noticed the error. Looking at the repo, I realized that Brew was pulling the latest tagged version, which is 0.0.3, but that the current version seems to be 0.0.4. I installed 0.0.4 and noticed that the same issue existed in both 'versions'

**Comment by Bazerghi** [comment]
BTW, the example .i2csshrc config file in my bug above is obviously missing valid spacing and is an invalid YAML format. I can assure you that the .i2csshrc config I'm using is valid YAML, has proper spacing, no tabs, etc.

**Comment by wouterdebie** [comment]
Thanks for this bug report! I found the bug and fixed it. This should be fixed in 0.0.7, which should now also be properly available in homebrew.
